tools/pd-simulator/simulator/cases/import_data.go

```diff
@@ -120,21 +112,28 @@ func newImportData() *Case {
 		regionTotal := regions.GetRegionCount()
 		totalLeaderLog := fmt.Sprintf("%d leader:", regionTotal)
 		totalPeerLog := fmt.Sprintf("%d peer:", regionTotal*3)
-		isEnd := true
+		isEnd := checkCount > uint64(getRegionNum())/10
+		var regionProps []float64
 		for storeID := uint64(1); storeID <= 10; storeID++ {
 			regions.GetStoreRegionCount(storeID)
 			totalLeaderLog = fmt.Sprintf("%s [store %d]:%.2f%%", totalLeaderLog, storeID, float64(regions.GetStoreLeaderCount(storeID))/float64(regionTotal)*100)
 			regionProp := float64(regions.GetStoreRegionCount(storeID)) / float64(regionTotal*3) * 100
-			if regionProp > 13.8 {
-				isEnd = false
-			}
+			regionProps = append(regionProps, regionProp)
 			totalPeerLog = fmt.Sprintf("%s [store %d]:%.2f%%", totalPeerLog, storeID, regionProp)
 		}
 		simutil.Logger.Info("import data information",
 			zap.String("table-leader", tableLeaderLog),
 			zap.String("table-peer", tablePeerLog),
 			zap.String("total-leader", totalLeaderLog),
 			zap.String("total-peer", totalPeerLog))
+		checkCount++
+		dev := 0.0
+		for _, p := range regionProps {
+			dev += (p - 10) * (p - 10) / 100
```

Comment:
will it still be 10 after region num is configured?

Reply:
It means if balanced, store each get 10 % of data, no matter what the total is.

Reply:
OK